packages/ui/src/components/ButtonWithDropdown.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, ReactNode } from "react";
+import { Button, ButtonGroup, ButtonProps } from "./Button.js";
+import { Dropdown, TriangleIcon } from "../index.js";
+import clsx from "clsx";
+
+type Props = ButtonProps & {
+  renderDropdown(options: { close(): void }): ReactNode;
+};
+
+export const ButtonWithDropdown: FC<Props> = ({ renderDropdown, ...props }) => {
+  return (
+    <ButtonGroup>
+      <Button {...props} />
+      {/* Button -> Dropdown order is important, because Button must be a direct child of ButtonGroup */}
+      <Button
+        noLayout
+        size={props.size}
+        theme={props.theme}
+        disabled={props.disabled}
+      >
+        <Dropdown render={renderDropdown} placement="bottom-end" fullHeight>
+          <div
+            className={clsx(
+              props.size === "small" ? "px-1.5" : "px-3",
+              "flex h-full items-center"
+            )}
+          >
+            <TriangleIcon
+              className="rotate-180"
```

Comment:
Triangle color is a bit too dark, especially on normal-sized buttons.
<img width="259" alt="Captura de pantalla 2023-11-07 a la(s) 16 31 37" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/a8064961-968e-4af2-9b6f-787197c2ba58">

I couldn't override it because the main `<Button>` component picks the color based on a combination of `disabled`, `theme`, hover state, etc. But maybe we could add `opacity-80` here.

Reply:
I wouldn't object to the ``opacity-80`` hack.